Corona-Warn-App/src/main/java/de/rki/coronawarnapp/risk/TraceLocationCheckInRisk.kt

```diff
@@ -4,6 +4,6 @@ import org.joda.time.LocalDate
 
 interface TraceLocationCheckInRisk {
     val checkInId: Long
-    val localDate: LocalDate
+    val localDateUtc: LocalDate
```

Comment:
A LocalDate doesn't have timezone information, if I'm not mistaken..

Reply:
The local data does not have a timezone anymore, that's correct. The conversion to local date uses a timezone, so I wanted to make it clear that the date here is derived using utc, as I would otherwise expect a local date to be derived using the local timezone.